cmd/boulder-wfe/main.go

```diff
@@ -102,6 +104,15 @@ func setupWFE(c config, logger blog.Logger, stats prometheus.Registerer, clk clo
 	return rac, sac, rns, npm
 }
 
+type errorWriter struct {
+	blog.Logger
+}
+
+func (ew errorWriter) Write(p []byte) (n int, err error) {
+	ew.Logger.Err(string(p))
```

Comment:
Might want to prefix this with something to identify that it's coming from inside net/http.

Reply:
Good idea.